src/scss/default/gaiden.scss

```diff
@@ -1,21 +1,11 @@
-@import 'node_modules/reset-css/reset.scss'; //sass-lint:disable-line clean-import-paths
+@import 'node_modules/reset-css/_reset'; //sass-lint:disable-line clean-import-paths
```

Comment:
Acho que nem precisa do `underscore`. 🤔

Reply:
tava dando not found, resolveu quando coloquei o underscore haha